Scripts/reset_db.py

```diff
@@ -0,0 +1,54 @@
+import sqlite3
+from typing import List
+
+#get list of emails from file
+with open('dummy_data/emails.txt','r') as e_File:
+    emailList: List = e_File.read().split('\n')
+    emailList = list(set(emailList))
+
+#get list of constraints from file
+with open('dummy_data/not_allowed.txt','r') as na_File:
+    forbiddenPairsRaw: List = na_File.read().split('\n')
+    forbiddenPairsRaw = list(set(forbiddenPairsRaw))
+
+print(forbiddenPairsRaw)
+#turn list of strings into list of tuples
+constraintList: List = []
+for i in range(len(forbiddenPairsRaw)):
+    a, b = forbiddenPairsRaw[i].strip("()").replace("'","").split(',')
+    constraintList.append((a,b))
+    try:
+        emailList.index(a)
+    except ValueError:
+        raise SystemExit("The name %r is in not_allowed.txt, but not in emails.txt" % a)

```

Comment:
Use f strings here! https://realpython.com/python-f-strings/

Reply:
This changed my life.